CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # CHANGELOG
 ## Release 2.11.5 (in development)
+### Enhancements
 
+- Make Loki pod resources configurable
```

Comment:
Could be good to add a doc section showing how this can be used to limit the memory, if you have time

Reply:
Right, ok

Reply:
Done